domain/src/Commands/DomainCommands.php

```diff
@@ -0,0 +1,1288 @@
+    $stats = [];
+    $totals = [];
+    foreach ($domains as $domain) {
+      $stats[$domain->id()] = [];
+      foreach ($entity_typenames as $name) {
+        $stats[$domain->id()][$name] = $this->enumerateDomainEntities($name, $domain->id(), DOMAIN_ACCESS_FIELD, TRUE);
+      }
+      $totals[$domain->id()] = array_reduce($stats[$domain->id()], function ($sum, $v){ return $sum + $v; }, 0);
+    }
+
+    $keys = [
+      'weight',
+      'name',
+      'hostname',
+      'scheme',
+      'status',
+      'is_default',
+      'domain_id',
+      'id',
+    ];
+    $rows = [];
+    /** @var \Drupal\domain\DomainInterface $domain */
+    foreach ($domains as $domain) {
+      $row = [];
+      foreach ($keys as $key) {
+        $v = $domain->get($key);
+        switch($key) {
+          case 'status':
+            if (($options['inactive'] && $v) || ($options['active'] && !$v)) {
+              continue 3; // switch, for, for
+            }
+            $v = !empty($v) ? dt('Active') : dt('Inactive');
+            break;
+          case 'is_default':
+            $v = !empty($v) ? dt('Default') : '';
+            break;
+        }
+
+        $row[$key] = Html::escape($v);
+      }
+      $row['membership'] = $totals[$domain->id()];
+      $rows[] = $row;
+    }
+    return new RowsOfFields($rows);
+  }
+
+  /**
+   * List general information about the domains on the site.
+   *
+   * @usage drush domain:info
+   *
+   * @command domain:info
+   * @aliases domain-info,dinf
+   *
+   * @return \Consolidation\OutputFormatters\StructuredData\PropertyList
+   * @field-labels
+   * count: All Domains
+   * count_active: Active Domains
+   * default_id: Default Domain ID
+   * default_host: Default Domain hostname
+   * scheme: Fields in Domain entity
+   * domain_access_entities: Domain access entities
+   * all_affiliate_support: All-affiliate support
+   * @list-orientation true
+   * @format table
+   * @throws \Drupal\domain\Commands\DomainCommandException
+   */
+  public function infoDomains() {
+    $domain_storage = $this->getStorage();
+
+    // Load all domains:
+    $all_domains = $domain_storage->loadMultiple(NULL);
+    $active_domains = [];
+    foreach ($all_domains as $domain) {
+      if ($domain->status()) {
+        $active_domains[] = $domain;
+      }
+    }
+
+    $keys = [
+      'count',
+      'count_active',
+      'default_id',
+      'default_host',
+      'scheme',
+    ];
+    $rows = [];
+    foreach ($keys as $key) {
+      $v = '';
+      switch($key) {
+        case 'count':
+          $v = count($all_domains);
+          break;
+        case 'count_active':
+          $v = count($active_domains);
+          break;
+        case 'default_id':
+          $v = $domain_storage->loadDefaultDomain()->getDomainId();
+          break;
+        case 'default_host':
+          $v = $domain_storage->loadDefaultDomain()->getHostname();
+          break;
+        case 'scheme':
```

Comment:
Technically, scheme can vary on a per-domain basis, so perhaps we need to list count by scheme? Or this should be `default_scheme`.

Reply:
Not sure I can deal with this without the dev environment.